CDTDatastore/CDTReplicator/CDTReplicator.h

```diff
@@ -258,4 +258,7 @@ initWithTDReplicatorManager:(nonnull TDReplicatorManager *)replicatorManager
  */
 @property (nullable, nonatomic, readonly) NSError *error;
 
+/** Exposed for testing. Returns the doc ID for the checkpoint document. */
+- (nonnull NSString *)remoteCheckpointDocID;
```

Comment:
We shouldn't have this exposed here even for testing, you can use the runtime to get the selector and then use `performSelector` to get the result.